dotnet/src/Connectors/Connectors.AzureOpenAI.UnitTests/Extensions/AzureOpenAIServiceCollectionExtensionsTests.cs

```diff
@@ -19,16 +20,26 @@ namespace SemanticKernel.Connectors.AzureOpenAI.UnitTests.Extensions;
 /// <summary>
 /// Unit tests for the service collection extensions in the <see cref="AzureOpenAIServiceCollectionExtensions"/> class.
 /// </summary>
-public sealed class AzureOpenAIServiceCollectionExtensionsTests
+public sealed class AzureOpenAIServiceCollectionExtensionsTests : IDisposable
 {
+    public AzureOpenAIServiceCollectionExtensionsTests()
+    {
+        this._httpClient = new HttpClient();
```

Comment:
This test is not ensuring that the client was being used. Use a private HttpMessageHandler in the client with an overrided `SendAsync` to ensure it was triggered after calling any of the methods in the `IChatCompletionService` interface.


```suggestion
        this._httpClient = new HttpClient();
```

Reply:
This test is updated and showing that the HttpClient is being used